src/api/uploads/BaseMultiput.js

```diff
@@ -16,7 +16,7 @@ const DEFAULT_MULTIPUT_CONFIG: MultiputConfig = {
     retries: 5 // How many times to retry requests such as upload part or commit. Note that total number of attempts will be retries + 1 in worst case where all attempts fail.
 };
 
-class BaseMultiput extends Base {
+class BaseMultiput extends BaseUpload {
```

Comment:
I don't think it should be extending from `BaseUpload`. Could probably merge `BaseUpload` into `PlainUpload`